src/box/lua/net_box.lua

```diff
@@ -1113,6 +1196,50 @@ local function check_eval_args(args)
     end
 end
 
+function stream_methods:stream()
```

Comment:
What is a purpose of this function? Maybe drop it?

Reply:
Since we want the stream to have the same methods as the connection, we done                                                                       local stream = setmetatable({
        _stream_id = stream_id,
        stream = stream_methods.stream,
        space = setmetatable({
            _stream = nil,
            _spaces = 0,
        }, stream_spaces_mt),
        _conn = self,
    }, { __index = self })
The problem is that in this case, the stream has a `stream` method inherited from the connection, which will not work correctly for the stream, we had to redefine it.

Reply:
According to our private discussion, i implement streams_methods table which contains only necessary for stream functions. Fixed